torchgeo/datasets/landcoverai.py

```diff
@@ -289,6 +290,35 @@ def __init__(
         self.batch_size = batch_size
         self.num_workers = num_workers
 
+    def on_after_batch_transfer(
+        self, batch: Dict[str, Any], batch_idx: int
+    ) -> Dict[str, Any]:
+        """Apply batch augmentations after batch is transferred to the device.
+
+        Args:
+            batch: mini-batch of data
+            batch_idx: batch index
+
+        Returns:
+            augmented mini-batch
+        """
+        try:
+            if self.trainer.training:  # type: ignore[union-attr]
```

Comment:
@isaaccorley you mean like this?

Reply:
Yeah that should do it. 

On a side note, we should make the augs be an input to SegmentationTask at some point so that users can define their own, although issues due to user error would be a pain.